TACC/Core-Styles
# feat: color demo + fixes

## Overview & Changes

Added Color Settings demo.

## Related

…

## Testing

1. Open http://localhost:3000/components/detail/color--default.
2. Open the other Color pages.
3. Verify Default & CMS use blue accent.
4. Verify Docs & Portal use purple accent.
5. Verify Default & CMS & Docs uses different set of info/warning/danger/success/extra colors than Portal.

## UI

### Full List

| cms | portal |
| - | - |
| ![all colors](https://github.com/TACC/Core-Styles/assets/62723358/fdba7420-0c79-4086-9590-49c87b94ea53) | ![localhost_3000_components_preview_color--portal](https://github.com/TACC/Core-Styles/assets/62723358/69bd5a27-ce72-4032-8557-67db547de7c2) |

### Demo UI

| default | cms |
| - | - |
| ![demo - default](https://github.com/TACC/Core-Styles/assets/62723358/7f45a5ac-8032-4e67-9ecb-609f7fa57d48) | ![demo - cms](https://github.com/TACC/Core-Styles/assets/62723358/d9508b20-32b4-4e92-8c0d-995571563b12) |

| docs | portal |
| - | - |
| ![demo - docs](https://github.com/TACC/Core-Styles/assets/62723358/a161d970-9e5f-47b2-9cfd-f41e3df187a5) | ![demo - portal](https://github.com/TACC/Core-Styles/assets/62723358/b35ba55f-5eb4-4c1e-ac23-22cbc3809ba6) |
